packages/@ourworldindata/grapher/src/core/grapher.scss

```diff
@@ -242,13 +242,4 @@ $zindex-controls-drawer: 150;
     .markdown-text-wrap__line:last-child {
         margin-bottom: 2px;
     }
-
-    a.dod-span {
```

Comment:
We no longer need to override the styles for links.

Reply:
As Marcel pointed out, we still need to add the styles to Grapher. Added a mixin for this.